helmfile/50-applications.yaml

```diff
@@ -158,7 +158,8 @@ releases:
     group: starboard
   chart: ./charts/starboard/starboard-operator-psp-rbac
   version: 0.1.0
-  installed: {{ .Values.starboard.enabled }}
+  installed: false
```

Comment:
The procedure to remove a helm release nowadays is to explicitly remove it via a migration snippet.

Reply:
You should also remove Falco PSP RBAC.